docs/readthedocs/source/doc/LLM/QuickStart/Install BigDL-LLM on Windows for Intel GPU.md

```diff
@@ -0,0 +1,157 @@
+
+<img src="./figs/fig4.png" alt="image-20240221105834031" style="zoom:20%;" />
+
+### Install conda
+
+We recommend using miniconda to create environment. Please refer to the [page](https://docs.anaconda.com/free/miniconda/) to install miniconda. 
+
+* Choose windows miniconda installer. Download and install. It takes a few minutes. 
+
+  <img src="./figs/fig5.png" alt="image-20240221110402278" style="zoom:20%;" />
+
+* After installation, open `Anaconda prompt` and create an environment by `conda create -n llm python=3.9 libuv` . 
+
+  > Note: if you encounter CondaHTTPError problem and fail to create the environment, please check the internet connection and proxy setting. You can define your proxy setting by `conda config --set proxy_servers.http your_http_proxy_IP:port` and `conda config --set proxy_servers.https your_https_proxy_IP:port`
+  >
+  > <img src="./figs/fig6.png" alt="image-20240221122852777" style="zoom:20%;" />
+
+### Install oneAPI 
+
+ Install oneAPI Base Toolkit with the help of pip. After ensuring  `conda` is ready, we can use `pip ` to install oneAPI Base Toolkit. 
+
+```bash
+pip install dpcpp-cpp-rt==2024.0.2 mkl-dpcpp==2024.0.0 onednn==2024.0.0
+```
+
+> If you encounter HTTP Timeout error, also check your internet and proxy setting in `pip.ini` file which is under "C:\Users\YourName\AppData\Roaming\pip"  folder. 
+
+When you successfully install oneAPI from pip, you will see similar thing from the Anaconda prompt command line. <img src="./figs/fig7.png" alt="image-20240221130508668" style="zoom:20%;" />
+
+### Install bigdl-llm
+
+1. Step 1: Run the commands below in Anaconda prompt. 
+
+```bash
+conda create -n llm python=3.9 libuv # Already done in "Install conda" section
+conda activate llm
+pip install dpcpp-cpp-rt==2024.0.2 mkl-dpcpp==2024.0.0 onednn==2024.0.0 # Already done in "Install oneAPI" section
+pip install --pre --upgrade bigdl-llm[xpu] -f https://developer.intel.com/ipex-whl-stable-xpu
+```
+
+
+2. Step 2: Now we can test whether all the components have been installed correctly. If we can import all the packages correctly following the python file below, then the installation is correct. 
+   ```python
+   import torch
+   import time
+   import argparse
+   import numpy as np
+   
+   from bigdl.llm.transformers import AutoModel,AutoModelForCausalLM
+   from transformers import AutoTokenizer, GenerationConfig
+   ```
+
+   Then we use phi-1.5 as an example to show how to run the model with bigdl-llm on windows. 
+   ```python
+   import torch
+   import time
+   import argparse
+   import numpy as np
+   
+   from bigdl.llm.transformers import AutoModel,AutoModelForCausalLM
+   from transformers import AutoTokenizer, GenerationConfig
+   
+   PHI1_5_PROMPT_FORMAT = " Question:{prompt}\n\n Answer:"
+   generation_config = GenerationConfig(use_cache = True)
+   
+   if __name__ == '__main__':
+       parser = argparse.ArgumentParser(description='Predict Tokens using `generate()` API for phi-1_5 model')

```

Comment:
Make this example as simple as possible without much code. 
- remove arg parse section (just arg values in code)
- remove timings